at-chuongphu/20180718084554_change_gender_user.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeGenderUser < ActiveRecord::Migration[5.2]
+  def up
+    User.where(gender: 'nam').update_all(gender: 0)
+    User.where(gender: 'nu').update_all(gender: 1)
+    User.where(gender: 'khac').update_all(gender: 2)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/af52c561d1515fcd022559e1290a2c239e9fe3d5/at-chuongphu/20180718084554_change_gender_user.rb#L5"></span>Rails/SkipsModelValidations: Avoid using <code>update_all</code> because it skips validations. (http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)</td>
    </tr>
  </tbody>
</table>